AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoModel.swift

```diff
@@ -276,12 +274,60 @@ class VideoModel: NSObject {
             }
         }
     }
+    
+    func addAllRemoteVideosInCurrentPageExceptUserPausedVideos() {
```

Comment:
looks like this function does not reflect user paused videos.

Reply:
This comment is still valid. if we call `removeAllRemoteVideos()` and then add videos of the current page back, what will happen to the previous user paused video, do we need to explicitly pause them after the subscription according to `userPausedVideoTileIds`? 

Also, as mentioned by Sang as well, I would vote +1 to change func name to `subscribeAllRemoteVideoxxx`/`unsubscribeAllRemoteVideos`.